configure.ac

```diff
@@ -39,7 +39,7 @@ AC_PATH_TOOL(RANLIB, ranlib)
 AC_PATH_TOOL(STRIP, strip)
 
 AM_PROG_CC_C_O
-AC_PROG_CC_C89
+AC_PROG_CC
 if test x"$ac_cv_prog_cc_c89" = x"no"; then
```

Comment:
(Fun that  `ac_cv_prog_cc_c89` is still present in `configure`, so it's correct to keep that check)